datafusion/sqllogictest/src/engines/conversion.rs

```diff
@@ -77,7 +77,21 @@ pub(crate) fn f64_to_str(value: f64) -> String {
     } else if value == f64::NEG_INFINITY {
         "-Infinity".to_string()
     } else {
-        big_decimal_to_str(BigDecimal::from_str(&value.to_string()).unwrap())
+        big_decimal_to_str(BigDecimal::from_str(&value.to_string()).unwrap(), None)
+    }
+}
+
+pub(crate) fn spark_f64_to_str(value: f64) -> String {
```

Comment:
this looks like a copy/paste of `f64_to_str` -- maybe we could just thread the spark flag down and avoid some duplication. Not necesary just a suggestion